initial_support_arm : 'psm_two'

# translation x, y, z, orientation, x, y, z, w, arm index, grasp index
# goal state for simulation
goal_state : [0.1008570548406, -0.0496507108570937, 0.139795535108907, -0.78803504976098, -0.101598131882979, -0.230585369007219, 0.561701849334595,
              0.079306624785935, -0.188628939931201, 0.208250726633585, -2.32970278776429, 0.884206725052175, 0.828382001719908,
             1.0, 147.0]

# SE3 bounds: x_min, x_max, y_min, y_max, z_min, z_max wr/t /world from
se3_bounds : [-0.105, 0.105, -0.06, 0.06, 0.13, 0.21]

arm_index_bounds : [1, 2]

object_name : 'needle_r'

max_distance : 100.0

planning_time : 500.0
